apps/bmd/src/components/ElectionInfo.tsx

```diff
@@ -78,6 +82,12 @@ const ElectionInfo = ({
                 </NoWrap>
               )}{' '}
               {ballotStyleId && <NoWrap>ballot style {ballotStyleId}</NoWrap>}
+              {showElectionHash && (
+                <React.Fragment>
+                  <br />
+                  {'Election ID: ' + electionHash.substring(0, 20)}
```

Comment:
This could just be `Election ID: {electionHash.substring(0, 20)}`, right? Also, I think we typically use just 10 when showing it for visual comparison (cc @benadida).